nbextensions/viewCell/widgets/viewCellWidget.js

```diff
@@ -1,1343 +1,1355 @@
-    './appParamsViewWidget',
-    'google-code-prettify/prettify',
-    'css!google-code-prettify/prettify.css',
-    'css!font-awesome.css',
-], (
-    $,
-    Promise,
-    Uuid,
-    JupyterNamespace,
-    Lang,
-    Runtime,
-    Events,
-    html,
-    Props,
-    Narrative,
-    BusEventManager,
-    NarrativeMethodStore,
-    Workspace,
-    PythonInterop,
-    utils,
-    Ui,
-    Fsm,
-    Spec,
-    AppParamsWidget,
-    AppParamsViewWidget,
-    PR
-) => {
-    'use strict';
-    const t = html.tag,
-        div = t('div'),
-        span = t('span'),
-        a = t('a'),
-        p = t('p'),
-        table = t('table'),
-        tr = t('tr'),
-        th = t('th'),
-        td = t('td'),
-        pre = t('pre'),
-        input = t('input'),
-        appStates = [
-            {
-                state: {
-                    mode: 'new',
-                },
-                ui: {
-                    buttons: {
-                        enabled: [],
-                        disabled: ['run-app'],
+define(
+    [
+        'jquery',
+        'bluebird',
+        'uuid',
+        'base/js/namespace',
+        'common/lang',
+        'common/runtime',
+        'common/events',
+        'common/html',
+        'common/props',
+        'common/jupyter',
+        'common/busEventManager',
+        'kb_service/client/narrativeMethodStore',
+        'kb_service/client/workspace',
+        'common/pythonInterop',
+        'common/cellUtils',
+        'common/ui',
+        'common/fsm',
+        'common/spec',
+        './appParamsWidget',
+        './appParamsViewWidget',
+        'google-code-prettify/prettify',
+        'css!google-code-prettify/prettify',
+    ],
+    (
+        $,
+        Promise,
+        Uuid,
+        JupyterNamespace,
+        Lang,
+        Runtime,
+        Events,
+        html,
+        Props,
+        Narrative,
+        BusEventManager,
+        NarrativeMethodStore,
+        Workspace,
+        PythonInterop,
+        utils,
+        Ui,
+        Fsm,
+        Spec,
+        AppParamsWidget,
+        AppParamsViewWidget,
+        PR
+    ) => {
+        'use strict';
+        const t = html.tag,
+            div = t('div'),
+            span = t('span'),
+            a = t('a'),
+            p = t('p'),
+            table = t('table'),
+            tr = t('tr'),
+            th = t('th'),
+            td = t('td'),
+            pre = t('pre'),
+            input = t('input'),
+            appStates = [
+                {
+                    state: {
+                        mode: 'new',
                     },
-                    elements: {
-                        show: [],
-                        hide: [
-                            'fatal-error',
-                            'parameters-group',
-                            'output-group',
-                            'parameters-display-group',
-                            'exec-group',
-                        ],
+                    ui: {
```

Comment:
line 65 is obviously different from line 64 in the old file. 🤦

Reply:
Prettier + Git = 🤯

Reply:
Just to verify, the only changes here were the CSS imports, right?